src/app/pages/tabs/tab3/tab3.page.html

```diff
@@ -1,10 +1,30 @@
 <app-page-header [config]="headerConfig"></app-page-header>
+
+<ion-toolbar>
+  <ion-segment value="mentor" swipeGesture="true" (ionChange)="segmentChanged($event)">
+    <ion-segment-button value="mentor">
+      <ion-label>Mentor</ion-label>
```

Comment:
Use translate string functionality. The strings should be added in the en.json file and used in pages.